suggestedimages/routes.py

```diff
@@ -2,36 +2,50 @@
     Blueprint, flash, redirect, render_template, request, url_for
 )
 import urllib
+from collections import namedtuple
+
 
 from . import search
 from .util import StrInLanguage
 from .locales import Locale
 
+LangOption = namedtuple("LangOption", "value label")
```

Comment:
Here shortening the variable name is ok. Still, it should be considered if there is any reason not to use LanguageOption instead.

Reply:
I agree with @stenvma81 , LanguageOption adds somewhat more readability plus you dont have to write shorten version.

Reply:
You're right it's unnecessary to abbreviate it and it's only used twice. Changed to LanguageOption.